feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -53,28 +99,84 @@ func TestMain(m *testing.M) {
 //    https://github.com/fullstorydev/grpcurl
 //
 
+// gNOIKillProcess kills a daemon on the DUT, given its name and pid.
+func gNOIKillProcess(ctx context.Context, t *testing.T, args *testArgs, pName string, pID uint32) {
+	gnoiClient := args.dut.RawAPIs().GNOI().Default(t)
+	killRequest := &gnps.KillProcessRequest{Name: pName, Pid: pID, Signal: gnps.KillProcessRequest_SIGNAL_TERM,
+		Restart: true}
+	killResponse, err := gnoiClient.System().KillProcess(context.Background(), killRequest)
```

Comment:
the killResponse has nothing interesting in it. I'd ignore the value, i.e.
_, err := gnoiClient.System()...

Reply:
> the killResponse has nothing interesting in it. I'd ignore the value, i.e. _, err := gnoiClient.System()...

For debuggability purposes I am logging killResponse. Is it ok to leave this as is?'

Reply:
No, the KillProcessResponse has nothing in it, so it will just be noise in the log that a viewer may mistake as something being wrong.

Reply:
Resolved in latest git commit